### ISSUE TITLE
Can fetch pkg support streaming?

### ISSUE THREAD

**Issue by lyekumchew** [syumai/workers#85]
When requesting a large file, it will cause `err: failed on promise: TypeError: Memory limit would be exceeded before EOF.`
``` Go
func main() {
	http.HandleFunc("/", func(w http.ResponseWriter, req *http.Request) {
		fetchReq, err := fetch.NewRequest(req.Context(), http.MethodGet, "http://lg.hkg.hosthatch.com/1GB.test", nil)
		if err != nil {
			w.WriteHeader(http.StatusInternalServerError)
			return
		}
		resp, err := fetch.NewClient().Do(fetchReq, nil)
		if err != nil {
			w.WriteHeader(http.StatusInternalServerError)
			fmt.Fprintf(w, "err: %v", err)
			return
		}
		defer resp.Body.Close()
		io.Copy(w, resp.Body)
	})
	workers.Serve(nil)
}
```

**Comment by syumai** [comment]
I'll check this later.

**Comment by syumai** [comment]
Perhaps it would be better to add an API to return the raw Response Object obtained from a fetch call.

**Comment by syumai** [comment]
Calling the `Read` method by `io.Copy` generates many CPU calls for large streams (maybe unacceptable even on a paid plan).
`io.Copy` called with ResponseWriter on the Go side and ReadableStream on the JavaScript side should use ReadableStream directly instead of calling the `Read` method.
I'll try to implement this feature.

**Comment by syumai** [comment]
@lyekumchew I've added changes related to this issue in #92.
Would you check if your code works with [workers@v0.23.1](https://github.com/syumai/workers/releases/tag/v0.23.1)?

**Comment by lyekumchew** [comment]
@syumai After trying https://github.com/syumai/workers/pull/93, it works.

**Comment by syumai** [comment]
@lyekumchew @aki-0421
This feature has been released as v0.23.2.
https://github.com/syumai/workers/releases/tag/v0.23.2
